### PR TITLE
PR for mobile version contact form

### PR DESCRIPTION
**Pull request by misikirdelelew**
In this PR I have done

- Contact form page
- styled the contact form based on the Figma design

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @MishDM , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🟥 

- [ ] You have submitted a project with an incomplete README file. Please do not delete the sections of this file. Fill in all of them except the sections marked **OPTONAL**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -242,21 +242,14 @@ <h2 class="contact-h2">Contact me</h2>
...
-                    <button type="submit" class="get-in-touch-btn
-                    ">Get It Touch</button>
-                    <div class="error-message">
-                        <small id="error"></small>
-                    </div>
+                    <button type="submit">Get It Touch</button>
                 </form>
-                <img src="/images/contact-ba-bar.png" alt="bar at the bottom" class="home-indicator">
+                <img src="./images/contact-ba-bar.png" alt="bar at the End">
             </article>
```
Comment (mckent05) [inline index.html:253]: - This section of your portfolio does not exactly look like what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
-- The background images are missing for this section. Please fix this 😄 
ethethte
-- Your submit button seems to be misaligned on bigger screens, please fix this 🙏 
ryskyk

File: css/style.css [hunk 2]
```diff
@@ -411,15 +416,11 @@ header {
 }
 
 .contact-form button:hover {
-  background-color: #817dfa;
+  background-color: #a3a0f6;
```
Comment (mckent05) [inline css/style.css]: - You have used the wrong hover color for your button, please fix this 🙏 .Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @MishDM, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -242,21 +242,14 @@ <h2 class="contact-h2">Contact me</h2>
...
                         placeholder="write your message in here" maxlength="500" required></textarea>
-                    <button type="submit" class="get-in-touch-btn
-                    ">Get It Touch</button>
-                    <div class="error-message">
-                        <small id="error"></small>
-                    </div>
+                    <button type="submit">Get It Touch</button>
                 </form>
-                <img src="/images/contact-ba-bar.png" alt="bar at the bottom" class="home-indicator">
+                <img src="./images/contact-ba-bar.png" alt="bar at the End">
```
Comment (TadesseDev) [inline index.html:252]: - [ ] Kudos on the changes you make @MishDM, But there are some minor details to be added following the required changes from the previous reviewer. 
> One is your button needs to be aligned right with the rest of the form elements (straight to the text area)
> The other is your form container needs to take the whole width of the mobile display.

### Figma design 
Screenshot 2023-01-18 214423

### Your design 
Screenshot 2023-01-18 214634

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @MishDM 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highligt
 - [ ] Good job making most of the changes requested by the previous code reviewer. ✅
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @MishDM  ,
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**